README.md

```diff
@@ -29,6 +29,7 @@ This package is a native addon, built with C++ code using `node-addon-api`, a C+
 * Node.js 14
 * Node.js 15
```

Comment:
Alan, could you remove Node.js 16 as well? Just to reduce the number of commits PRs, makes most sense to do it here.

Reply:
@markdirish I've added another commit to the branch to remove version 15. Let me know if I need to rebase, but perhaps you can squash and merge.

Reply:
Looks good. Could you now make the DCO bot happy? There are instructions in this link: https://github.com/markdirish/node-odbc/pull/315/checks?check_run_id=12195617013

I believe it should be:

1. Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
2.  In your local branch, run: `git rebase HEAD~2 --signoff`
3. Force push your changes to overwrite the branch: `git push --force-with-lease origin patch-1`